### ISSUE TITLE
Is the document wrong for IGListSectionController?

### ISSUE THREAD

**Issue by PhilCai1993** [Instagram/IGListKit#432]
In `IGListSectionController.h`, the comment for 

```
@property (nonatomic, weak, nullable) id <IGListScrollDelegate> scrollDelegate;
```
is 
```
/**
 An object that handles display events for the section controller. Can be `nil`.

 @return An object that conforms to `IGListDisplayDelegate` or `nil`.

 @note You may wish to return `self` if your section controller implements this protocol.
 */
```
which is exactly the same as
```
@property (nonatomic, weak, nullable) id <IGListDisplayDelegate> displayDelegate;
```

Or is there some difference?
Thanks~

**Comment by rnystrom** [comment]
@PhilCai1993 I'm loving your deep dives. Yes! This is wrong. It should be updated and describe the scrolling delegate instead.

**Comment by PhilCai1993** [comment]
😄 We have added `IGListKit` into our Podfile.
And I have to share it to my partners, so I take a a deep look at it.

**Comment by Sherlouk** [comment]
Closed by 6e00787
